src/screens/Login/styles.ts

```diff
@@ -0,0 +1,48 @@
+import { StyleSheet } from "react-native";
+import { colors, grid, montserrat } from "@app/utils";
+import { RFValue } from "react-native-responsive-fontsize";
+
+interface StyleProps {
+  hasKeyboard: boolean;
+}
+
+export const useStyles = ({ hasKeyboard }: StyleProps) =>
+  StyleSheet.create({
+    container: {
+      flex: 1,
+    },
+    background: {
+      flex: 1,
+      alignItems: "center",
+    },
+    content: {
+      alignItems: "center",
+      width: "100%",
+      padding: grid.genericSpacing,
+    },
+    logo: {
+      marginVertical: grid.genericSpacing,
+      alignItems: "center",
+      justifyContent: "center",
+    },
+    logoText: {
+      fontSize: RFValue(38),
+      textTransform: "uppercase",
+      textAlign: "center",
+      fontFamily: montserrat.bold,
+      color: colors.secondary,
+    },
+    cardContainer: {
+      position: "absolute",
+      width: "100%",
+      top: hasKeyboard ? "15%" : "40%",
```

Comment:
Is there a way to add animations/transition? Bit janky the way to shifts up.

Reply:
I need to look into this further because animations are handled by a separate react native component